content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst

```diff
@@ -0,0 +1,38 @@
+=================
+Payment providers
+=================
+
+The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,
+adding additional payment providers for your customers is strongly recommended. Odoo supports a
+multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
+for your website.
+
+.. seealso::
+   :doc:`../../../../applications/finance/payment_providers`
+
+Configuration
+-------------
+
+To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration
+--> Payment Providers`. From here, :guilabel:`Activate` the payment providers you wish to have
+available on your shop, and configure them according to your needs.
+
+Alternatively, you can access **payment providers** via :menuselection:`Website --> Configuration
+--> Settings`. In the :guilabel:`Shop - Payment` section, you can :guilabel:`Configure SEPA Direct
+Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
+:guilabel:`Authorize.net` payment provider, the
+:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
+same menu.
+
+If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure
+the :guilabel:`Paypal - Express Checkout` feature in this menu.
```

Comment:
The "Paypal express checkout" feature doesn't exist (yet).

@pierre-pde You promised to remove it!